fleet_vehicle_inspection/i18n/pt_BR.po

```diff
@@ -578,6 +595,15 @@ msgstr "Linha de Inspeção do Veículo"
 msgid "Vehicles Inspections"
 msgstr "Inspeções de Veículos"
 
+#~ msgid "Inspection Line Result "
+#~ msgstr "Resultado da Linha de Inspeção "
+
+#~ msgid "NOK"
+#~ msgstr "NOK"
+
+#~ msgid "OK"
+#~ msgstr "OK"
```

Comment:
We should avoid changing translations files directly. They should be translated using [OCA translation platform](https://odoo-community.org/page/translate)

> https://odoo-community.org/blog/the-oca-blog-1/post/translations-platform-81:
Important: always translate using Weblate, and never push changes to .po files manually. This simple rule will avoid all merge conflicts, as all .pot file changes are done by Travis, and Weblate is the master of all .po files.

Reply:
Done